lgc/include/lgc/state/AbiMetadata.h

```diff
@@ -66,13 +66,13 @@ enum PipelineType : unsigned {
 
 // Hardware shader stage
 enum class HardwareStage : unsigned {
-  Ls = 0, ///< Hardware LS stage
-  Hs,     ///< Hardware hS stage
-  Es,     ///< Hardware ES stage
-  Gs,     ///< Hardware GS stage
-  Vs,     ///< Hardware VS stage
-  Ps,     ///< Hardware PS stage
-  Cs,     ///< Hardware CS stage
+  Ls = 0, // Hardware LS stage
+  Hs,     // Hardware hS stage
+  Es,     // Hardware ES stage
+  Gs,     // Hardware GS stage
+  Vs,     // Hardware VS stage
+  Ps,     // Hardware PS stage
+  Cs,     // Hardware CS stage
```

Comment:
Is there a reason for removing the "///<"?

While we don't actually use it particularly well, the idea here is that "///<" initiate a "document comment" that refers to the left-hand-side on the same line. It feels slightly better to keep it, and in any case this is just an unnecessary change.

I think the same applies to some other enums.

Reply:
> Is there a reason for removing the "///<"?

We use the mix of "///<" and "//" in comments. I think it is better for us to clarify the usage style in the contribution doc and coding style doc. Otherwise, this is very confusing for new comers to follow. In PAL coding style, only interface files that are used by clients adopt the doxygen style ///< and other source files keep normal style //.